var/spack/repos/builtin/packages/oracle-instant-client/package.py

```diff
@@ -106,3 +108,11 @@ def install(self, spec, prefix):
 
         install_tree(join_path('odbc', 'help'), prefix.doc)
         install_tree(join_path('sdk', 'sdk', 'include'), prefix.include)
+
+        # Fix broken symlinks: `zipfile` does not correctly
+        # handle symlinks when unpacking
+        for fn in glob.glob(join_path(prefix, 'lib', '*.so*')):
+            m_type, m_subtype = relocate.mime_type(fn)
+            if m_type == 'text':
+                orig_path = open(fn).read()
+                force_symlink(orig_path, fn)
```

Comment:
Apologies for taking so long to look at this: https://github.com/spack/spack/pull/30407 was merged in the last 5 days and restores the use of system `unzip`, do you find that this patch is still required?